rcljava/src/main/cpp/org_ros2_rcljava_node_NodeImpl.cpp

```diff
@@ -250,3 +252,59 @@ Java_org_ros2_rcljava_node_NodeImpl_nativeCreateTimerHandle(
   jlong jtimer = reinterpret_cast<jlong>(timer);
   return jtimer;
 }
+
+JNIEXPORT void JNICALL
+Java_org_ros2_rcljava_node_NodeImpl_nativeGetPublishersInfo(
+  JNIEnv * env, jclass, jlong handle, jstring jtopic_name, jobject jpublishers_info)
+{
+  rcl_node_t * node = reinterpret_cast<rcl_node_t *>(handle);
```

Comment:
Consider checking if null. I guess we'll throw an error on L281 if node is null, but it's probably not a bad idea to check anyways.

Reply:
Done in https://github.com/osrf/ros2_java/pull/26/commits/0024d68a62ed3809ff5cbaaccce06467f4e00f04.